scripts/_compile/index.js

```diff
@@ -112,21 +118,23 @@ async function compileScss () {
 
 async function compileScripts () {
     console.log('SRC: Compiling JavaScript');
-
-    // TODO: convert to npm script
-    await exec(`yarn rollup`, { cwd: CONFIG.root }, function(error, stderr){
+    await exec(`yarn gulp bundle --prod`, { cwd: CONFIG.root }, function(error, stdout, stderr){
         if (error) {
             console.error(`exec error: ${error}`);
             return;
         }
         console.log(`stderr: ${stderr}`);
+        console.log(`stdout: ${stdout}`);
```

Comment:
With the change of callback args, the console output now displays warnings that can be useful for development.  There were a couple of things I had to tweak/upgrade to correct potential issues that were brought to light with this additional information.